src/main/java/com/contentful/java/cma/ModuleAssets.java

```diff
@@ -100,7 +100,7 @@ public String delete(String spaceId, String assetId) {
 
   /**
    * Fetch all Assets from a Space.
-   *
+   * <p>
```

Comment:
Yes, it is called [javadoc](http://www.oracle.com/technetwork/articles/java/index-137868.html), the default way of writing comments in java.